src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -73,6 +73,9 @@ export const NewAnnotationForm = ({
 	const geneticModifierRelationTerms = useControlledVocabularyService('disease_genetic_modifier_relation');
 	const [uiErrorMessages, setUiErrorMessages] = useState({});
 	const areUiErrors = useRef(false);
+	const optionalFields = (["Asserted Genes", "Asserted Allele", "Negated", "With", "Related Notes", "Experimental Conditions", "Experiments", "Genetic Sex",
+							"Disease Qualifiers", "SGD Strain Background", "Annotation Type", "Genetic Modifier Relation", "Genetic Modifiers"]);
```

Comment:
Everything looks good. Only change that I suggest is that you don't need parenthesis around this array.

Reply:
Sure, will remove and update